modules/determining-upgrade-viability-conditiontype.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * updating/index.adoc
+
+:_content-type: CONCEPT
+[id="Understanding_clusteroperator_conditiontypes_{context}"]
+= Understanding cluster operator condition types
+
+The status of cluster operators is revealed with condition type, informing you of potential impacts to upgrading your cluster. 
+The following condition types have consistent wording across all ClusterVersion and ClusterOperator conditions. Individual components may also have additional condition types with component-specific language.
+
+The condition type, as well as additional informatin about your operator can be retrieved in either YAML or JSON format through the `oc get clusterversion -o` command:
+
+[source,terminal]
+----
+$ oc get clusterversion -o yaml
+----
+
+
+* Available: 
+The condition with type `Available` indicates that the operators are functional and available in the cluster. If the status is `False`, then the status indicates that at least one part of the operand is non-functional and the condition requires an administrator to intervene.
+
+* Progressing:
+The condition with type `Progressing` indicates that the operator is actively rolling out new code, propagating configuration changes, or otherwise moving from one steady state to another. 
++
+Operators should not report as `Progressing` when they are reconciling a previous known state. If the observed cluster state has changed and the operator is reacting to it, then the status should report back as `True`, since it is moving from one steady state to another.
```

Comment:
The "should" here are aimed at operator developers who are learning about how their operator should handle a new-to-the-dev condition.  But now that we're in customer/consumer docs, we can assume all the operator developers are on board, and we can say "Operators do not..." and "the status will report back...".  And similar for other "should" in this doc.  The assertions will be aspirational, and we'll sometimes fail to deliver, but anyone who sees us not delivering can open a case or bug, and we'll adjust the operators to do a better job living up to the docs.